dockstore-common/src/main/java/io/dockstore/common/DescriptorLanguage.java

```diff
@@ -80,6 +80,12 @@ public boolean isRelevantFileType(FileType type) {
         public boolean isRelevantFileType(FileType type) {
             return super.isRelevantFileType(type) || type == FileType.DOCKSTORE_SERVICE_OTHER;
         }
+    },
+    IPYNB("ipynb", "Jupyter IPYNB notebook", FileType.DOCKSTORE_IPYNB, null, false, false, Set.of("ipynb"), false, false) {
```

Comment:
second last parameter implies that this won't support hosted, intentional?

Reply:
Yeah, if eventually, we either provide a local notebooks environment, or we figure out a way to launch our hosted content to a notebook platform, we'll probably toggle this to `true`.